test/test_nn.py

```diff
@@ -8483,6 +8483,7 @@ def test_affine_grid_error_checking(self):
         with self.assertRaisesRegex(NotImplementedError, "affine_grid only supports 4D and 5D sizes"):
             F.affine_grid(theta, torch.Size([1, 1, 2, 2, 2, 2]), align_corners=False)
 
+    @skipIfRocm
```

Comment:
Is there an issue tracking those? Or how can we make sure those will get re-enabled once migration is done?
P.S. Does it make sense to add something like `@skipIfRocmVersionLessThan(4, 2)` so that we can continue supporting say 4.1 and 4.2 in parallel?

Reply:
@malfet We have internal JIRA for tracking to re-enable these tests.
For @skipIfRocmVersionLessThan(4, 2), will need to submit a separate PR for that. Will discuss with @jeffdaily and come back on that.